manifests/4.1/elasticsearch-operator.v4.1.0.clusterserviceversion.yaml

```diff
@@ -68,6 +69,10 @@ spec:
 
   keywords: ['elasticsearch', 'jaeger']
 
+  replaces: elasticsearch-operator.v4.1.0
+  # this will eventually be populated for any subsequent z-stream fixes
+  skips: []
```

Comment:
This isn't needed for z-streams, since the `skipRange` will take priority

Reply:
@ecordell so I leave `replaces` but remove the empty `skip`?

Reply:
removed both items